### PR TITLE
Include skipped message when using GTEST_SKIP macro

### PR DESCRIPTION
**Pull request by neyrojasj**
This change will include the gtest skipped message into the result.xml output file. This PR is solving issue: https://github.com/pytest-dev/pytest-cpp/issues/69

### CODE REVIEW

**Review by nicoddemus — CHANGES_REQUESTED** [review]
Thanks @neyrojasj for the PR, greatly appreciate it!

A couple of requests:

Please add or update one of the test cases to make sure the skipped message is appearing to the user?
Update the CHANGELOG with the new support.

Thanks!

File: src/pytest_cpp/google.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def run_test(self, executable, test_id, test_args=(), harness=None):
                 if failures:
                     return [GoogleTestFailure(x) for x in failures], output
                 elif skipped:
-                    pytest.skip()
+                    [pytest.skip(x) for x in skipped]
```
Comment (nicoddemus) [inline src/pytest_cpp/google.py:101]: This will really only show the first failure, as `pytest.skip()` raises an exception. Is that what you expect?

**Review by nicoddemus — APPROVED** [review]
Awesome @neyrojasj! 

Left a quick question.

File: src/pytest_cpp/google.py (hunk 1 above)
Comment (neyrojasj) [reply src/pytest_cpp/google.py:101]: @nicoddemus Yes, since the tests are run one by one,  that array should only contain one element. So, we are okay 😄

Comment (nicoddemus) [reply src/pytest_cpp/google.py:101]: Oh I know that, just pointing out that even if a test emitted (say) 3 messages, we only show one skipped message.

We could decide to use this instead:

```python
pytest.skip("\n".join(skipped))
```

Comment (nicoddemus) [inline src/pytest_cpp/google.py:101]:
```suggestion
                    pytest.skip("\n".join(skipped))
```

Comment (nicoddemus) [reply src/pytest_cpp/google.py]: Went ahead and applied that 👍

**Comment by nicoddemus** [comment]
`pytest-cpp 2.1.0` released, thanks!

File: src/pytest_cpp/google.py (hunk 1 above)
Comment (neyrojasj) [reply src/pytest_cpp/google.py]: Awesome @nicoddemus, thanks 😄

**Pull request merged**